src/db/sqlite/sql.rs

```diff
@@ -13,7 +13,7 @@ INSERT INTO users (email, password, is_admin) VALUES (?1, ?2, ?3);
 ";
 
 pub(crate) const UPDATE_USER: &str = "
-UPDATE table SET 
+UPDATE users SET 
     email = ?2,
     password = ?3,
     is_admin = ?4,
```

Comment:
```suggestion
    is_admin = ?4
```

Reply:
Not sure if this change makes it easier for you to commit and merge, or I should open a separate PR

Reply:
I don't beleive it does. I think a second PR would work

Reply:
Ah, that's unfortunate; opened a new PR #47

Reply:
No problem, sorry for not answering earlier your previous PR. I have been quite busy lately. I'm looking forward to working on this again.

Reply:
No worries whatsoever; completely understandable. This is a fun and good project and it has shaved off quite a lot of time that I would've otherwise have to spend getting a login system for my personal side-project working, so just wanted to say thanks, and contribute back minor changes that I needed to get it working on my thing. :)